src/Sarif.Driver/Sdk/CommandBase.cs

```diff
@@ -95,11 +95,9 @@ public static HashSet<string> CreateTargetsSet(IEnumerable<string> targetSpecifi
 
                 if (Uri.TryCreate(specifier, UriKind.RelativeOrAbsolute, out Uri uri))
                 {
-                    if (uri.IsAbsoluteUri && (uri.IsFile || uri.IsUnc))
-                    {
-                        normalizedSpecifier = uri.LocalPath;
-                    }
+                    normalizedSpecifier = uri.GetFileName();
```

Comment:
>GetFileName [](http://example.com/codeflow?start=46&length=11)

Here's another clean-up item, we need to call the GetFilePath and GetFileName helpers consistently across the code base (to properly process things like relative URLs).